portfolio/src/main/webapp/script.js

```diff
@@ -61,28 +61,60 @@ function filterCourseDisplay(courseType = 'course') {
 }
 
 /**
- * Fetch data from server and display on DOM.
+ * Fetch comments from server and display on DOM.
  */
-function getAndDisplayComments() {
+function loadComments() {
   fetch('/list-comments')
   .then(response => response.json())
   .then((data) => {
-    // Display each comment in a block with a like button .
+    const display = document.getElementById('content');
     for (let comment of data) {
-      const commentContainer = document.createElement('div');
-      commentContainer.class = 'comment-container';
-      commentContainer.innerHTML = '';
-      commentContainer.innerHTML += 'Posted by: ' + comment.name + '<br>';
-      commentContainer.innerHTML += comment.content + '<br>';
-      commentContainer.innerHTML += comment.numLikes + ' likes<br>';
-      commentContainer.innerHTML += 'Posted at ' + comment.timestamp + '<br>';
-      // TODO: use methods to create button.
-      const likeButtonHTML = '<form action="/add-like" method="POST">' + 
-        '<input type="hidden" name="comment-key" value="' + comment.key + '" />' + 
-        '<input type="submit" value="Like" class="btn btn-danger btn-sm" /></form>';
-      commentContainer.innerHTML += likeButtonHTML;
-      commentContainer.innerHTML += '<br><br>';
-      document.getElementById('content').appendChild(commentContainer);
+      display.appendChild(createCommentElement(comment));
     }
   });
-}
\ No newline at end of file
+}
+
+/**
+ * Create HTML display for one comment.
+ */
+function createCommentElement(comment) {
+  const commentContainer = document.createElement('div');
+
+  const commentText = document.createElement('p');
+  commentText.innerHTML += 'Posted by: ' + comment.name + '<br>';
+  commentText.innerHTML += comment.content + '<br>';
+  commentText.innerHTML += comment.numLikes + ' likes<br>';
+  commentText.innerHTML += 'Posted at ' + comment.timestamp;
```

Comment:
Optional: create different elements for every one of these (i.e. commentName, commentContent, commentNumLikes, commentTimestamp). That way you wont need the line breaks and then you have the option to style each piece of text differently (i.e. bold the 'Posted by: name' or make the comment content bigger text)